### ISSUE TITLE
Heroku Install Not Receiving Events

### ISSUE THREAD

**Issue by Vandivier** [sendgrid/eventkit-rails#4]
Deployed Event Kit Rails via the Heroku button. I didn't touch a line of code.
Install went quick and smooth. The result was the setup GUI webpage. I completed the set up and my information was correctly saved. Next stop: Authentication, followed by the main page GUI which displays the latest collected events.

Problem: It never registered any events. I used the testing function from SendGrid's back-end and it returned 401 errors, but I verified the credentials.

I contacted SendGrid support. They verified that 401 errors were not occurring, although they were being displayed. They claim that 500 errors were the errors which were actually occurring.

One of the guys from support tried to quickly execute a Heroku deployment and he was able to duplicate the errors.

In short, the Rails Event Kit is not registering events when deployed via Heroku.
